packages/shared/src/scheduler.ts

```diff
@@ -625,15 +625,13 @@ async function scheduleLookupCall<
 
   await client.dev.newBlock()
 
-  // Check if `parachainInfo` pallet exists
+  // In the collectives chain, dispatch of lookup calls does not work at present.
+  // Fix: https://github.com/polkadot-fellows/runtimes/pull/614
+  // Check if `parachainInfo` pallet exists to distinguish between collectives and other chains.
```

Comment:
no the best way to detect chain. could just an argument passing or check api.system.chain.
but this is a short workaround to be removed after next upgrade so it is fine

Reply:
I overengineered this one - you're right, this could just be an extra arg in tests config.